modules/programs/firefox.nix

```diff
@@ -300,7 +300,7 @@ in
           if isDarwin then
             cfg.package
           else if versionAtLeast config.home.stateVersion "19.09" then
-            cfg.package.override (old: { cfg = old.cfg or {} // fcfg; })
+            cfg.package.override (old: old // { cfg = old.cfg or {} // fcfg; })
```

Comment:
Does this actually change anything? Looks like [the `makeOverridable` implementation](https://github.com/NixOS/nixpkgs/blob/8e84cafcd9350ce11e3f630003b859d4c70a483a/lib/customisation.nix#L74) performs `origArgs // newArgs` anyway, therefore even with the existing code any other settings should not be lost.

Reply:
Thanks.

I suppose people are having issues because they are using a wrapped (or unwrapped?) version of Firefox